cytoxnet/dataprep/featurize.py

```diff
@@ -1,2 +1,101 @@
 # featuriziers in DC
 # mol descriptors not. maybe query?
+
+import pandas as pd
+import deepchem
+import rdkit
+
+
+def molstr_to_Mol(dataframe, strcolumnID='InChI String'):
+    """
+    Converts DataFrame column containing the string representations of
+    molecules and add a corresponding column of Mol objects.
+
+    Parameters
+    ----------
+    dataframe: DataFrame containing a column with string representations
+        of molecules
+    strcolumnID: label for the column containg the string representations for
+        converstion, default='InChI String'- can be changed for the user's
+        standard column name
+
+    Returns
+    -------
+    DataFrame with additional column containing Mol objects for each molecule,
+        column label is 'Mol'
+
+    """
+
+    # get the column containting the string representation
+    # check whether column title contains "inche" or "smiles"
+    if 'inchi' in strcolumnID.lower():
+        # convert InChI to Mol
+        for inchi in column:
+            mol_obj = rdkit.Chem.inchi.MolFromInchi(inchi)
+            #append to array first here probably
+    elif 'smiles' in strcpolumnID.lower():
+        # convert SMILES to Mol
+        for smiles in column:
+            mol_obj = MolFromSmiles(smiles)
+            # probably append to an array
```

Comment:
Might be worth at some point having the type (smiles or inchi) be specifiable by the user, parsing the string will fail if for some reason the name is not in the column name